redash/handlers/dashboards.py

```diff
@@ -150,7 +150,7 @@ def get(self, dashboard_slug=None):
         :>json string widget.updated_at: ISO format timestamp for last widget modification
         """
         dashboard = get_object_or_404(
-            models.Dashboard.get_by_slug_and_org, dashboard_slug, self.current_org
+            models.Dashboard.get_by_id_or_slug_and_org, dashboard_slug, self.current_org
```

Comment:
On a second thought, it's actually safer to have a different route for this 🤔, it can cause problems with existing `/dashboard/:slug` urls.

Reply:
yeah, see comment.